pkg/scanner/transformer.go

```diff
@@ -17,6 +21,7 @@ func (c *systemClock) Now() time.Time {
 }
 
 type Transformer interface {
+	ToClairLayers(req harbor.ScanRequest, manifest distribution.Manifest) []clair.Layer
 	Transform(artifact harbor.Artifact, source clair.LayerEnvelope) harbor.ScanReport
```

Comment:
What about renaming `Transform` to `ToHarborScanReport` so that it match pair with `ToClairLayers` method ?

Reply:
Good point. Changed. Thx